### PR TITLE
[VEN-2049]: Add UNI market to the Core pool

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Add UNI market to the Core pool

Resolves VEN-2048

### CODE REVIEW

**Review by chechu — COMMENTED** [review]
Testnet VIP lgtm

File: vips/vip-195/vip-195.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      {
+        target: VUNI,
+        signature: "transfer(address,uint256)",
+        params: [VTREASURY, INITIAL_VTOKENS],
+      },
+
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDT, parseUnits("10000", 18), NORMAL_TIMELOCK],
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply vips/vip-195/vip-195.ts]: Done  https://github.com/VenusProtocol/vips/pull/97/commits/ae26eeaafe3e3b7294b49a36fe4502e54719d0ea

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]

File: src/vip-framework/checks/interestRateModel.ts [hunk 2]
```diff
@@ -3,12 +3,11 @@ import { BigNumber, Contract, utils } from "ethers";
 import { ethers } from "hardhat";
 
 import RATE_MODEL_ABI from "../abi/il_rateModel.json";
-import VTOKEN_ABI from "../abi/il_vToken.json";
 
 const BLOCKS_PER_YEAR = BigNumber.from(10512000);
 
 export async function checkInterestRate(
-  vTokenAddress: string,
+  RATE_MODEL: string,
```
Comment (kkirka) [inline src/vip-framework/checks/interestRateModel.ts:10]:
```suggestion
  rateModel: string,
```

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/195

File: src/vip-framework/checks/interestRateModel.ts (hunk 2 above)
Comment (GitGuru7) [reply src/vip-framework/checks/interestRateModel.ts]: [d90bce4](https://github.com/VenusProtocol/vips/pull/97/commits/d90bce4d84cff1ca8e4f6d8182649ce226423aa8)

**Pull request merged**